components/automate-ui/src/app/modules/infra-proxy/json-treeable/json-treeable.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, Input, ElementRef, OnChanges } from '@angular/core';
+import * as jsonTree from '../../../../app/page-components/json-tree/vendor/json-tree';
+
+// TODO:eng-ex: This might do better as a service or module.
+//              Unclear because there is styling on the javascript but no template.
+@Component({
+  selector: 'app-json-treeable',
```

Comment:
I am assuming you mean treetable not treeable ? Please correct the selector here, the class name, and the file name.